src/hooks/useTeamsBreadcrumbs.ts

```diff
@@ -0,0 +1,61 @@
+import { useMemo } from 'react';
+import { setTeamColor } from '~v5/common/TeamReputationSummary/utils';
+import useColonyContext from './useColonyContext';
+
+export const useGetSelectedTeamFilter = () => {
+  const { colony } = useColonyContext();
+  const { domains } = colony || {};
+
+  const urlParams = new URLSearchParams(window.location.search);
+  const team = urlParams.get('team');
+
+  return useMemo(
+    () =>
+      domains?.items.find(
+        (domain) => domain?.nativeId === parseFloat(team || '0'),
+      ),
+    [domains?.items, team],
+  );
```

Comment:
It would be good to add an `effect` here when someone comes to the page with an invalid team id then the team id is removed from the URL